include/z64cutscene_commands.h

```diff
@@ -22,8 +22,10 @@
  */
 
 #ifdef __GNUC__
+#define CS_FLOAT(ieee754bin, f) (f)
 #define CMD_F(a) {.f = (a)}
```

Comment:
Not the biggest fan of this macro taking two numbers, its pretty confusing to see two numbers being passed into a macro and only one to be used depending on the context. I tbh would prefer if hex was used in all cases with comments to describe the float representation

I guess this is a solution to the "portability" problem that was mentioned before?

Reply:
Yeah we have the option to just write the hex as well rather than the macro

Personally I don't mind either

Reply:
changed to "hex"

Reply:
do the zapd cutscene files have the float representation as a comment? I see the non-zapd ones dont. feel like that is necessary at least for the the cam commands where the floats need to be intelligible

Reply:
They don't
I can probably PR another "hex-comment" option to ZAPD, sounds simple enough for me to do

Reply:
alright
I feel like we may have not had enough input on this-- I like just hex, but the macro with dual input might be a good compromise for those who feel strongly about them being in float form

Reply:
FTR I don't mind changing back

It's not particularly difficult either:
- the macro definitions
- the zapd argument in `extract_assets.py`
- the similar hardcoded argument in `csdis.py`
- run `csdis_re.py`

Reply:
changed back to "both" (`CS_FLOAT(hex, f)` macro) after having too much trouble trying to have a tool encode floats on build like asm-processor did